<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AMQPFooleryWidget</class>
 <widget class="QWidget" name="AMQPFooleryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>531</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>637</width>
    <height>531</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RabbitMQ Client</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Connection</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="ServerAddressLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="VHostLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>VHost:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ServerAddressLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>221</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="VHostLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>221</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectPushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>101</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectPushButton_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>101</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>620</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Configuration</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="ExchangeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Exchange:</string>
    </property>
   </widget>
   <widget class="QLabel" name="RoutingKeyLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Routing Key:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ExchangeLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>341</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="RoutingKeyLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>341</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BindPushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>101</width>
      <height>86</height>
     </rect>
    </property>
    <property name="text">
     <string>Bind</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Queue Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="QueueNameLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>341</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>620</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Message</string>
   </property>
   <widget class="QPushButton" name="ACKPushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>250</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ACK</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="AutoACKCheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto ACK</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearMessageWindowPushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Message Window</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>601</width>
      <height>211</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>ClearMessageWindowPushButton</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
